src/main/java/com/bravo/user/dao/model/User.java

```diff
@@ -18,6 +18,26 @@
   @Column(name = "id")
   private String id;
 
+  public void setFirstName(String firstName) {
```

Comment:
These getters/setters are not necessary since the [Lombok](https://projectlombok.org/) `@Data` annotation provides them.